x-pack/plugins/asset_manager/server/routes/assets.ts

```diff
@@ -127,3 +186,13 @@ export function assetsRoutes<T extends RequestHandlerContext>({ router }: SetupR
     }
   );
 }
+
+function toArray<T>(maybeArray: T | T[] | undefined): T[] {
```

Comment:
if we move this to a `utils.ts` file, we could reuse it in `get_all_related_assets.ts` - it also has a `toArray` function